Hello Microverse Project Pull Request

IvonneBenitesRodriguez opened the pull request:
**About the project:**
- This project is about a small site with a stylesheet and an index.html file.
- The project has only a header with a h1 inside.
- The project has the configuration for Webhint and StyleLint tools usage.
- A new branch with its own issue was created to add all the configuration and code before adding to the main branch.

**Why these tools are important?**
- These tools help developers to identify, modify and deliver a clean and efficient code.

   **what was the most complex part?**
  -   As it is the first time I see Linters tool, I was a bit confused. But now, with the practice I know that 
       it is important to add these tools to each project.

IvonneBenitesRodriguez committed:
Create: file named index.html

IvonneBenitesRodriguez committed:
Style: add stylesheet file

IvonneBenitesRodriguez committed:
Style: update stylesheet to add style to h1

IvonneBenitesRodriguez committed:
Update: add header and h1 to index.html file

IvonneBenitesRodriguez committed:
Chore: add .gitignore file

IvonneBenitesRodriguez committed:
Feat: add .github/workflows folder to store linters file

IvonneBenitesRodriguez committed:
Feat: add .hintrc file to config linters

IvonneBenitesRodriguez committed:
Add package-lock.json file for linters config

IvonneBenitesRodriguez committed:
Add package.json file to continue config linters

IvonneBenitesRodriguez committed:
Feat: add .stylelintrc.json to identify errorsin stylesheet

IvonneBenitesRodriguez committed:
Feat: update package-lock.json

IvonneBenitesRodriguez committed:
Feat: update package.json file

IvonneBenitesRodriguez committed:
Style: modify all changes required by Stylelint

IvonneBenitesRodriguez committed:
Update README.md

This README.md  has the information required to set up the tool linters in the project.

tresorsawasawa commented on README.md:
```diff
@@ -1 +1,70 @@
+    
+- [📝 License](#license)
+  - This project is [MIT](./LICENSE) licensed.
+
```
- [ ] Good job trying to use the Microverse README template as required for the project. However, I noticed that you didn't follow the syntax and you miss some sections(e.g.: Aknowledgement,...) like from the orignal template. I suggest you update again to looks similar to the [required template ](https://github.com/microverseinc/readme-template) and customize it according to your project information. You can look at this **[example to get an idea of how to proceed](https://github.com/HtetWaiYan7191/math-magicians/blob/development/README.md)**.

IvonneBenitesRodriguez replied:
Hello 👋🏾 @tresorsawasawa 
I thank you for your review, 
I already made all the changes you suggested, and updated my README.md file according to my project information but following the syntax of the Microverse README template, can you please verify it  🙏🏿 . 
Since , sadly the system does not allow me to re-submit the project.
Thank you,

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @IvonneBenitesRodriguez 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

IvonneBenitesRodriguez committed:
Update README.md

I modified the README.md to correct some mistakes. 
The README.md is more specified and contains relevant commands for the linters configuration.

IvonneBenitesRodriguez committed:
Update README.md

Update the README.md to give structure and more relevant information to the project.

IvonneBenitesRodriguez committed:
Update README.md

Update README.md to modify the presentation and structure for the project.

AbelG101 commented on README.md:
```diff
@@ -1 +1,226 @@
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
- [ ] Hey Ivonne, I really appreciate how well you documented each section of the readme I just have a tiny suggestion on the "License" section here in the text below the MIT link which reads as "_NOTE: we recommend using the MIT license..." is a section description to guide the author on what type of license is advised but this should be removed from the final draft so I kindly suggest you remove it as it makes your project look unprofessional 👶
- This is the entire text you should remove 👇
  > 
_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

IvonneBenitesRodriguez replied:
Hello @AbelG101,
Very nice to meet you 😊 
Thank you for your suggestion, I appreciate it.
 I modified and updated the part you suggested me.
Best regards,

AbelG101 reviewed (changes requested):
![](https://media.giphy.com/media/7SyGkO0kQWCfnUC47Y/giphy.gif)


# Hi @IvonneBenitesRodriguez 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Professionally documented readme ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

IvonneBenitesRodriguez committed:
Update README.md

README.md file about this project updated.

Munsa1 reviewed (approved):
Hi @IvonneBenitesRodriguez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 
Made All Requested Changes ✔️ 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
